modules/@apostrophecms/user/index.js

```diff
@@ -52,7 +52,20 @@ module.exports = {
     publishRole: 'admin',
     viewRole: 'admin',
     showPermissions: true,
-    relationshipSuggestionIcon: 'account-box-icon'
+    relationshipSuggestionIcon: 'account-box-icon',
+    scrypt: {
+      // These are the defaults. If you choose to pass
```

Comment:
Can be a basis for documentation or the docs can just link to the scrypt docs, while mentioning that you need not set `maxmem` yourself.